src/SprykerEco/Yves/Payone/Theme/default/views/klarna/klarna.twig

```diff
@@ -0,0 +1,18 @@
+{% extends model('template') %}
+
+{% define data = {
+    form: form,
+} %}
+
+{% block body %}
+    {% include molecule('payone-klarna', 'Payone') with {
+        data: {
+            form: data.form,
+            billingAddressData: vars.billing_address_data,
+            customerData: vars.customer_data,
+        },
+        attributes: {
+            'klarna-pay-methods': vars.widget_pay_methods | json_encode(),
+        },
```

Comment:
- Please get values via the `data` object.

- Usually, data comes in `_view` object from BE side. @abitskil

Reply:
- Refactored via using of the `data` object.
- Actually it comes from `view` object but in this case it implemented in `payment.twig` view of the `CheckoutPage` module on the project level and it's not an issue for the current refactoring.